drivers/gpu/drm/vc4/vc4_firmware_kms.c

```diff
@@ -1613,13 +1620,29 @@ static const struct drm_encoder_funcs vc4_fkms_encoder_funcs = {
 	.destroy = vc4_fkms_encoder_destroy,
 };
 
+static void vc4_fkms_display_power(struct drm_encoder *encoder, bool power)
+{
+	struct vc4_fkms_encoder *vc4_encoder = to_vc4_fkms_encoder(encoder);
+	struct vc4_dev *vc4 = to_vc4_dev(encoder->dev);
+
+	struct mailbox_display_pwr pwr = {
+		.tag1 = {RPI_FIRMWARE_SET_DISPLAY_POWER, 8, 0, },
+		.display = vc4_encoder->display_num,
+		.state = power ? 1 : 0
```

Comment:
It's an annoying nit-pick, but a adding a comma at the end of the line would match all the other properties and the normal Linux style (the intention being to minimise inter-line dependencies).